### PR TITLE
add config for migrations and seeds, create user entity

### PR DESCRIPTION
**Pull request by nadyrbek97**
https://trello.com/c/gpNktPFf/1-base-app-nestjs-app

### CODE REVIEW

File: backend/src/app.module.ts [hunk 1]
```diff
@@ -1,15 +1,15 @@
-import { Module } from '@nestjs/common';
+import {forwardRef, Module} from '@nestjs/common';
```
Comment (seregaskirev) [inline backend/src/app.module.ts]: let's use eslint and prettier, can you implement it?

**Review by seregaskirev — CHANGES_REQUESTED** [review]
let's include eslint and prettier to the project

File: backend/src/app.module.ts (hunk 1 above)
Comment (nadyrbek97) [reply backend/src/app.module.ts]: thanks , done

**Review by seregaskirev — APPROVED** [review]
👍

**Pull request merged**